src/Components/SelfInterview/utils.ts

```diff
@@ -0,0 +1,19 @@
+import { Question } from "../../Apis/types";
+
+export const sortMethod = (questions: Question[], switchNumber: number) => {
+    let result = questions.sort((a: any, b: any) => {
+        if (a.question < b.question) return -1;
+        if (a.question > b.question) return 1;
+        return 0;
+    });
```

Comment:
realizing now this is sub-optimal, should have done this sort but then possibly iterated over each string and compared the char at each index and if one was longer than we stop comparing at the shorter question.

Reply:
or a localeCompare could have worked